scripts/js/lib/api/htmlToMd.test.ts

```diff
@@ -482,6 +482,26 @@ test("parse deprecations warnings", async () => {
 </Admonition>
 `);
 });
+// ------------------------------------------------------------------
+// Handle version added derivative admonition
+// ------------------------------------------------------------------
```

Comment:
Often, this type of comment is really helpful. Here, it's a bit noisy and I'd remove it because the test for versionadded is part of the section we already have for "admonitions"